packages/form-handlers-mailchannels-test/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "@webstudio-is/form-handlers-mailchannels-test",
+  "private": true,
+  "version": "0.0.0",
+  "scripts": {
+    "deploy": "wrangler publish src/index.ts",
+    "dev": "wrangler dev src/index.ts"
```

Comment:
Am I right, if I execute pnpm run dev at tge root this will be executed too?

Reply:
Yes. Fixing